### PR TITLE
docs: add more concrete example cases of making PRs

### PR DESCRIPTION
**Pull request by IshitaTakeshi**
Signed-off-by: IshitaTakeshi <ishitah.takeshi@gmail.com>

## Description

I thought that contributors might be curious that their proposals deserve to open Q&A and pull requests, so I added more concrete example cases

https://github.com/autowarefoundation/autoware/discussions/271

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kenji-miyake — COMMENTED** [review]
The content looks good to me. I'll leave the style review to Lalith-san. :pray:

File: docs/contributing/index.md [hunk 1]
```diff
@@ -83,6 +83,11 @@ You can submit pull requests for small changes such as:
 - Fixing warnings detected by compilers or analysis tools
 - Making small changes to a single package
 
-If your pull request is a large change not covered by the examples above, discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).
+If your pull request is a large change such as
+
+- Adding a new feature to Autoware
+- Adding a new page to the documentation
+
+discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).
```
Comment (LalithVipulananthan) [inline docs/contributing/index.md:91]: I think that it is better to provide examples at the end (this also avoids fragmenting the first sentence). How about this instead?

```suggestion
If your pull request is a large change, discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).

Examples of large changes include:
- Adding a new feature to Autoware
- Adding a new documentation page or section
```

Comment (IshitaTakeshi) [reply docs/contributing/index.md]: Looks good!

**Comment by IshitaTakeshi** [comment]
@LalithVipulananthan I pushed the commit suggestion button but it does not seem to provide the Signed-off-by line. Could you commit and force-push?

**Comment by kenji-miyake** [comment]
You can manually approve it.
https://github.com/autowarefoundation/autoware-documentation/pull/99/checks?check_run_id=6345816255
image

**Review by LalithVipulananthan — APPROVED** [review]
Approving commit to allow merge.

**Pull request merged**